migrations/1719503528757_add-variants.sql

```diff
@@ -0,0 +1,13 @@
+-- Up Migration
+CREATE TABLE variants (
+    id SERIAL NOT NULL,
+    unique_id TEXT NOT NULL,
+    author_id TEXT NOT NULL,
+    organization_id TEXT NOT NULL,
+    timestamp TIMESTAMP NOT NULL,
+    properties JSONB NOT NULL DEFAULT '{}'::JSONB
+);
+ALTER TABLE variants ADD PRIMARY KEY (id);
+
```

Comment:
- add an index on `unique_id` after alter table.
`CREATE INDEX variants_unique_id_idx ON variants (unique_id);`

- migrations/1719503528757_add-variants.sql should be the last file in migrations folder, update the prefix number to be bigger than `1719944573861_remove-unused-nih-ned-id.sql`

Reply:
added index creation clause and new file created with `npm run migrate create add variants`